### DISCUSSION TITLE
Issue with Auto Sequence and Sudden Motor Stop During Landing in Crazyflie

### DISCUSSION THREAD

**Discussion by amitx64** [bitcraze/discussions#1499] [Q&A]
I'm working on an auto sequence with Crazyflie, and it's performing the auto formation as expected. However, I've encountered an issue when triggering the landing at a height of 0.5 meters. At this point, the motors suddenly stop, and the drone falls instead of performing a smooth landing. 

I've double-checked my sequence setup and landing triggers, but I'm not sure why this is happening. Could anyone help me troubleshoot this issue?

**Comment by knmcguire** [comment]
Are you using any of the python autonomous examples? Also are you using a stock crazyflie with the latest support firmware and do you have the console output after reconnecting after such a crash?

**Reply by amitx64** [reply]
@knmcguire I’m not sure why, but when I retest by changing the landing position to somewhere else, it lands perfectly on the ground. It might be due to the `LocoNode` blind spot.

**Reply by knmcguire** [reply]
In that sense, it might be that the crazyflie thinks it's lower than it actually is. With loco positioining the z-axis state estimation is less accurate especially when the antenna's are not placed at an ideal location.

**Reply by amitx64** [reply]
@knmcguire yeah! absolutely... that's what i am thinking of

**Reply by amitx64** [reply]
@knmcguire the volume that I used is `X: 5.3m, Y: 3m, Z: 2.2m`

**Reply by knmcguire** [reply]
If you have the crazyflie standing on the floor and just look at the z-axis of your state estimate's position, does it think it is above the ground?
